src/app/zap-templates/zcl/data-model/chip/time-synchronization-cluster.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <struct name="TimeZoneType">
+    <item name="Offset" type="INT32S" min="-43200" max="50400"/>
+    <item name="ValidAt" type="INT64U"/> 
+    <item name="Name" type="CHAR_STRING" length="64"/>
+  </struct>
+
+  <struct name="DstOffsetType">
+    <item name="Offset" type="INT32S"/>
+    <item name="ValidStarting" type="INT64U"/> 
+    <item name="ValidUntil" type="INT64U"/>
+  </struct>
+
+
+  <enum name="Granularity" type="ENUM8">
```

Comment:
Why is this not matching the spec's `GranularityEnum`?

Reply:
I saw that in some other clusters `Enum` suffix is omitted, but I will change it to match the spec.